.travis/test-coverage.sh

```diff
@@ -3,6 +3,23 @@
 PROFILE_OUT=$PWD/profile.out
 ACC_OUT=$PWD/acc.out
 
+latestRelease=$(
+	git ls-remote --tags https://github.com/golang/go.git 'go*' \
+	| egrep 'go[1-9]+\.[0-9]+\.[0-9]+$' \
+	| sed -e 's|^.*tags/||g' \
+	| sort -V \
+	| tail -n 1
+)
+echo Latest Go releast: $latestRelease
+
+currentVersion=$(go version | cut -d' ' -f3)
+echo Current Go version: $currentVersion
+
+if [[ "$currentVersion" != "$latestRelease" ]]; then
```

Comment:
perhaps this piece of code should be modified to only test for the 'Major.Minor' version and not the 'Patch' one (as travis may lag a bit before providing the latest patch version to the build bots.) ?

Reply:
Done